### ISSUE TITLE
steampipe cache is backed by memory or disk?

### ISSUE THREAD

**Issue by zfLQ2qx2** [turbot/steampipe#2305]
I'm setting up Steampipe as service running under AWS ECS. For resource allocation purposes, is Steampipe query caching backed by RAM or disk?

**Comment by e-gineer** [comment]
The results cache is stored in memory.

BTW, I assume you've found our [Using containers](https://steampipe.io/docs/managing/containers) doc?

Hope that helps. Would love to learn more about what you are building?

**Comment by zfLQ2qx2** [comment]
@e-gineer That is helpful! I had overlooked the --foreground option. I'm actually using the Fargate flavor of AWS ECS so I can't do the mounts described. There also appears to be no way to direct logging to the console which is the, primary method of collecting logs in Fargate, so instead I've tuned it to errors only. The other issue I ran into is the postgres engine in Steampipe is not staticly linked and requires glibc so instead of Alpine linux I've had to resort to Ubuntu. Depending on how much use this gets I might invest time in doing a local build on Alpine. Overall the idea is to allow querying without giving away the credentials being used to access the data sources or making people learn the underlying APIs, and hopefully this will facilitate creating reports.

**Comment by e-gineer** [comment]
Makes sense @zfLQ2qx2 ... appreciate your feedback on the log to stdout instead of files ... For all it's power, Fargate has some interesting limitations which we've hit previously too.

Would definitely appreciate any further feedback, issues or PRs you can share 🚀 

You may also find [Steampipe Cloud](https://steampipe.io/docs/cloud/overview) an interesting option if you want centralized credential management with dashboard and query access for other users.
